recipes/libde265/all/conanfile.py

```diff
@@ -39,6 +40,11 @@ def configure(self):
         if self.settings.compiler.cppstd:
             tools.check_min_cppstd(self, 11)
 
+    def validate(self):
+        if self.options.sse:
+            if self.settings.os == "Macos" and self.settings.arch == "armv8":
+                raise ConanInvalidConfiguration("Option 'libde265:sse' with value True is not supported for Macos/armv8")
+
```

Comment:
sse is supported on x86 and x86_64 only.

Reply:
Sounds very reasonable. Thanks!